galley/pkg/config/schema/codegen/collections.go

```diff
@@ -126,10 +126,10 @@ func camelCase(n string, sep string) string {
 
 // Is c an ASCII lower-case letter?
 func isASCIILower(c byte) bool {
-	return 'a' <= c && c <= 'z'
+	return c >= 'a' && c <= 'z'
 }
 
 // Is c an ASCII digit?
 func isASCIIDigit(c byte) bool {
-	return '0' <= c && c <= '9'
+	return c >= '0' && c <= '9'
```

Comment:
nit: arguable stylistic choice :)

Reply:
I agree... We can `//nolint` or turn off the yodastyle check. I would have just disabled it, but it was already enabled, seems the upgrade just made it stricter. Any preference?

Reply:
My preference would be to disable the style check. But make sure @geeknoid is on-board.

Reply:
I am beaten by this kind of check.

Reply:
@hzxuzhonghu just to clarify, do you think we should disable this check?

Reply:
I donot feel string about this orders. Think if we check two vars equal, i would rather use this style 
`if 10 == a`  instead of `if a = 10` in c .

Reply:
The former style is more readable IMO. AFAICT, the motivation for yodastyle is to avoid inadvertently mixing up assignment and equality in languages like C. That shouldn't be necessary in golang. I'd vote for disabling the check.